src/System.Data.SqlClient/src/Interop/ILocaleApiHelper.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Concurrent;
+using System.Data.Common;
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Data
+{
+    internal interface ILocaleApiHelper
```

Comment:
for interop folder, we will move this to locale folder and LocaleInterop should be rename with something else. as interop call only for windows.

Ideally this locale functionality should be located into the same folder.
There shouldn't be separated between unix / windows implementation, as we look at functionality.

and this interface will be used for windows as well, to encapsulate between unix/ windows implementation